continuous-integration/windows/create_bdist.py

```diff
@@ -0,0 +1,13 @@
+from common import repo_root, run_process, install_pip_dependencies
```

Comment:
Could we also rename this file?

Reply:
Renamed these files to "create_binary_distribution.py" and "install_binary_distribution.py".